src/components/GradesView/GradesView.scss

```diff
@@ -76,49 +21,21 @@
   }
 }
 
-.filter-group {
-  .grade-filter-inputs {
-    .percent-group {
-      display: inline-block;
-      .form-group, .pgn__form-group {
-        width: 115px;
-        display: inline-block;
-      }
-      .input-percent-label {
-          margin-top: 22px;
-          margin-left: 5px;
-          margin-right: 5px;
-      }
-    }
-  }
-  .grade-filter-action {
-    text-align: right;
-  }
-}
-
-.mb-85 {
-    margin-bottom: 85px;
-}
-
 .modal-dialog {
     max-width: 1000px;
 }
 
-.wrap-text-in-cell {
-    overflow-wrap: break-word;
-    word-wrap: break-word;
-}
-
```

Comment:
Is text-wrap equivalent? I'm not sure.

Reply:
Actually `text-break` is equivalent to this.

```css
.text-break {
  word-break: break-word !important;
  overflow-wrap: break-word !important;
}
```